src/jit/lower.cpp

```diff
@@ -2960,8 +2988,43 @@ GenTree* Lowering::CreateFrameLinkUpdate(FrameLinkAction action)
     return storeInd;
 }
 
-// this is the lowering equivalent of genPInvokeMethodProlog
-// This creates the code that runs at the start of every method that has pinvoke calls
+
+//------------------------------------------------------------------------
+// InsertPInvokeMethodProlog: Create the code that runs at the start of
+// every method that has PInvoke calls.
+//
+// Initialize the TCB local and the InlinedCallFrame object. Then link ("push")
+// the InlinedCallFrame object on the Frame chain. The layout of InlinedCallFrame
+// is defined in vm/frames.h. See also vm/jitinterface.cpp for more information.
+// The offsets of these fields is returned by the VM in a call to ICorStaticInfo::getEEInfo().
+//
+// The (current) layout is as follows:
```

Comment:
A reasonable request, although looking at the VM sources can be very enlightening. Of course, we always use getEEInfo. I'll think about tweaking this in a subsequent change.